src/components/common/Typography/styles.ts

```diff
@@ -0,0 +1,29 @@
+import { css } from "styled-components";
+
+export const dpNormalize = css`
+  p {
+    margin-top: 0;
+    white-space: pre-wrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  p:first-child,
+  ol:first-child,
+  ul:first-child {
+    margin-top: 0;
+  }
+
+  ol, ul {
+    padding-inline-start: 20px;
+  }
+
+  img {
+    width: 100%;
+    height: auto;
+  }
+
+  a, a:hover {
```

Comment:
Minor: From a UI/UX experiance, if something is clickable then feeding that back to the user with a colour change, under or what not seems reasonable? Does this monotone color make it harder for the end user?

Reply:
In my opinion, it's not critical. But it's a great point. I created the story to do it.
https://app.shortcut.com/deskpro/story/156256/app-sdk-move-common-components-to-app-sdk